### DISCUSSION TITLE
Can a user provide annual land use land cover (or pft) as an input to fixed biogeography with no competition?

### DISCUSSION THREAD

**Discussion by ftzahura** [NGEET/fates#1094] [Q&A] [answered]
Hi all,

According to the FATES documentation the fixed biogeography mode reads pft from the surface data gridcells. I am wondering if in the no competition mode, users have the ability to define the annual PFTs for each year throughout the simulation. In ELM, there is an option to provide LULC timeseries. Does FATES offer similar functionality? I have seen some discussions on implementing lulc change for no competition. I am curious to learn whether these discussions have materialized into implemented features in FATES.

Thank you,
Faria

**Comment by ckoven** [comment, marked as answer]
Hi @ftzahura -- not yet. As you say, the V1 Land Use change PR allows for updates to the land use type, but doesn't work for no competition mode because it doesn't update the prescribed PFT (landcover) weightings based on the land use change. I am currently working with @glemieux on a second phase that will handle what you are talking about, the branch for which you can see at  https://github.com/ckoven/fates/compare/luh2...ckoven:fates:luh2_nocomp_merge. This is in active development so we will hopefully have results to share soon.

**Reply by ftzahura** [reply]
Awesome! Thank you so much for the update.
